pkg/cli/agent.go

```diff
@@ -37,7 +37,7 @@ type Agent struct {
 
 	DefaultAudioURL string `usage:"The default URL for the translation agent to use" default:"https://api.openai.com/v1/audio" env:"CLICKY_CHATS_AUDIO_SERVER_URL"`
 
-	APIURL      string `usage:"URL for API calls" default:"https://api.openai.com/v1/chat/completions" env:"CLICKY_CHATS_SERVER_URL"`
+	APIURL      string `usage:"URL for API calls" default:"http://localhost:8080/v1/chat/completions" env:"CLICKY_CHATS_SERVER_URL"`
```

Comment:
`ToolRunnerBaseURL` should also change if we are making this change.

Reply:
Done!